scripts/new_project.kts

```diff
@@ -4,7 +4,15 @@ object NewProject {
 
     private const val ARGUMENT_DELIMITER = "="
     private const val DOT_SEPARATOR = "."
+    private const val HELP_MESSAGE = """
+Run kscript new_project.kts to create a new project with the following arguments:
+    package-name=   New package name (i.e. com.example.package)
+    app-name=       New app name (i.e. MyApp, "My App")
+
+Example: kscript new_project.kts package-name=co.myproject.example app-name="My Project"
+    """
```

Comment:
Let's the format raw string to make it organized but you need to update the code a bit:
- In line `71` We need to call `.trimIndent()`-> `val argumentError = "ERROR: Invalid argument name: \n${HELP_MESSAGE.trimIndent()}"`
- And line `75` -> `message = HELP_MESSAGE.trimIndent()`
```suggestion
    private const val HELP_MESSAGE =
        """
            Run kscript new_project.kts to create a new project with the following arguments:
                package-name=   New package name (i.e. com.example.package)
                app-name=       New app name (i.e. MyApp, "My App")
            
            Example: kscript new_project.kts package-name=co.myproject.example app-name="My Project"
        """
```

Reply:
Or should we use `private val` and call` trimIndent()` when declaring the variable so we don't have to trim indent on each usage 🤔 

```
    private val HELP_MESSAGE =
        """
            Run kscript new_project.kts to create a new project with the following arguments:
                package-name=   New package name (i.e. com.example.package)
                app-name=       New app name (i.e. MyApp, "My App")
            
            Example: kscript new_project.kts package-name=co.myproject.example app-name="My Project"
        """.trimIndent()
```

Reply:
Good to me too 👍